specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/examples/SuccessfulAnalyzeConversationTranscriptJobsTaskStatusRequest.json

```diff
@@ -0,0 +1,79 @@
+{
+  "parameters": {
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "api-version": "2022-04-01-preview",
+    "Endpoint": "{Endpoint}",
+    "jobId": "c0f2a446-05d9-48fc-ba8f-3ef4af8d0b18"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "createdDateTime": "2020-10-01T15:00:45Z",
+        "displayName": "Redacting PII from transcribed audio",
+        "expirationDateTime": "2020-10-03T15:01:03Z",
+        "jobId": "c0f2a446-05d9-48fc-ba8f-3ef4af8d0b18",
+        "lastUpdateDateTime": "2022-01-25T15:01:03Z",
+        "status": "succeeded",
+        "tasks": {
+          "completed": 1,
+          "failed": 0,
+          "inProgress": 0,
+          "total": 1,
+          "items": [
+            {
+              "kind": "TranscriptionPiiResults",
+              "taskName": "Transcription PII",
+              "lastUpdateDateTime": "2022-01-25T15:01:03Z",
+              "status": "succeeded",
+              "results": {
+                "conversations": [
+                  {
+                    "id": "conv-01",
+                    "conversationItems": [
+                      {
+                        "id": "turn-01",
```

Comment:
Let's be consistent in our examples. In IRD its just `"id": "1"`.

Reply:
ok